seatunnel-connectors-v2/connector-iotdb/src/main/java/org/apache/seatunnel/connectors/seatunnel/iotdb/source/IoTDBSource.java

```diff
@@ -77,8 +77,13 @@ public void prepare(Config pluginConfig) throws PrepareFailException {
                 );
             }
         }
-        SeaTunnelSchema seatunnelSchema = SeaTunnelSchema.buildWithConfig(pluginConfig);
-        this.typeInfo = seatunnelSchema.getSeaTunnelRowType();
+        if (pluginConfig.hasPath(SeaTunnelSchema.SCHEMA.key())) {
```

Comment:
Schema is a required key of connector, so if the user does not configured it connector should throw exception. You can refer to #3947. Thanks!